addon/components/drop-zone.js

```diff
@@ -128,6 +128,30 @@ export default Ember.Component.extend({
   },
 
   getDropzoneOptions() {
+    const onDragEnterLeaveHandler = function(dropzoneInstance) {
+      const onDrag = ( element => {
+        let dragCounter = 0;
+
+        return {
+          enter(event) {
+            event.preventDefault();
+            dragCounter++;
+            element.classList.add('dz-drag-hover');
+          },
+          leave() {
+            dragCounter--;
+
+            if (dragCounter === 0) {
+              element.classList.remove('dz-drag-hover');
+            }
+          }
+        };
+      }).call(dropzoneInstance.element);
```

Comment:
This should be `call(this, dropzoneInstance.element)` and is the root cause of https://github.com/FutoRicky/ember-cli-dropzonejs/issues/42